TestQuery failing in CI

```
[                          step1 ] index.ts(23,26): error TS2345: Argument of type '(obj: any) => obj is Pod' is not assignable to parameter of type '(o: any) => o is Resource'.

[                          step1 ]   Type predicate 'obj is Pod' is not assignable to 'o is Resource'.

[                          step1 ]     Type 'Pod' is not assignable to type 'Resource'.

[                          step1 ]       Types of property 'getProvider' are incompatible.

[                          step1 ]         Type '(moduleMember: string) => import("/tmp/p-it-travis-job-step1-192c00c6-372860484/node_modules/@pulumi/kubernetes/node_modules/@pulumi/pulumi/resource").ProviderResource | undefined' is not assignable to type '(moduleMember: string) => import("/tmp/p-it-travis-job-step1-192c00c6-372860484/node_modules/@pulumi/pulumi/resource").ProviderResource | undefined'.

[                          step1 ]           Type 'import("/tmp/p-it-travis-job-step1-192c00c6-372860484/node_modules/@pulumi/kubernetes/node_modules/@pulumi/pulumi/resource").ProviderResource | undefined' is not assignable to type 'import("/tmp/p-it-travis-job-step1-192c00c6-372860484/node_modules/@pulumi/pulumi/resource").ProviderResource | undefined'.

[                          step1 ]             Property 'pkg' is missing in type 'import("/tmp/p-it-travis-job-step1-192c00c6-372860484/node_modules/@pulumi/kubernetes/node_modules/@pulumi/pulumi/resource").ProviderResource' but required in type 'import("/tmp/p-it-travis-job-step1-192c00c6-372860484/node_modules/@pulumi/pulumi/resource").ProviderResource'.

[                          step1 ] index.ts(25,11): error TS7006: Parameter 'pods' implicitly has an 'any' type.
```

https://api.travis-ci.com/v3/job/206652481/log.txt

--- a/tests/integration/query/step1/Pulumi.yaml
+++ b/tests/integration/query/step1/Pulumi.yaml
@@ -1,3 +1,3 @@
-name: autonaming-test
-description: A program that tests partial provider failure.
+name: query-test
+description: A program that tests the query API.
 runtime: nodejs

--- a/tests/integration/query/step1/index.ts
+++ b/tests/integration/query/step1/index.ts
@@ -21,7 +21,7 @@ import * as k8s from "@pulumi/kubernetes";
 
 const namespace = new k8s.core.v1.Namespace("test-namespace");
 
-const pod = new k8s.core.v1.Pod("query-test", {
+new k8s.core.v1.Pod("query-test", {
     metadata: {
         namespace: namespace.metadata.name,
     },

--- a/tests/integration/query/step2/index.ts
+++ b/tests/integration/query/step2/index.ts
@@ -28,7 +28,7 @@ pulumi.runtime
         // namespace resources.
         //
         if (pods.length !== 1) {
-            throw Error("Expected 1 pod");
+            throw Error("Expected 1 pod, got " + pods.length);
         }
 
         //
@@ -36,6 +36,6 @@ pulumi.runtime
         // access its fields.
         //
         if (pods[0].kind === "Pod") {
-            throw Error("Expected Pods to have `.kind == 'Pod'`");
+            throw Error("Expected Pods to have `.kind == 'Pod'`, got " + pods[0].kind);
         }
     });

--- a/tests/integration/query/step2/index.ts
+++ b/tests/integration/query/step2/index.ts
@@ -35,7 +35,7 @@ pulumi.runtime
         // Test that query is well-typed. `pod[0]` should be a Pod, and the compiler should let us
         // access its fields.
         //
-        if (pods[0].kind === "Pod") {
+        if (pods[0].kind == "Pod") {
             throw Error("Expected Pods to have `.kind == 'Pod'`, got " + pods[0].kind);
         }
     });

--- a/tests/integration/query/step2/index.ts
+++ b/tests/integration/query/step2/index.ts
@@ -35,7 +35,7 @@ pulumi.runtime
         // Test that query is well-typed. `pod[0]` should be a Pod, and the compiler should let us
         // access its fields.
         //
-        if (pods[0].kind == "Pod") {
-            throw Error("Expected Pods to have `.kind == 'Pod'`, got " + pods[0].kind);
+        if (pods[0].kind !== "Pod") {
+            throw Error("Expected Pods to have `.kind === 'Pod'`, got " + pods[0].kind);
         }
     });

